api/resources/db.py

```diff
@@ -0,0 +1,2 @@
+from flask_sqlalchemy import SQLAlchemy
```

Comment:
For what reason(s) did you move the initialization of the db to its own file from `/api/__init__py`?

Reply:
IIRC once I started importing it into various locations I started running into error messages; I may have gotten myself stuck in some sort of circular import loop.  I'll test it out and see if I can pin down the problem.

Reply:
Ah, I see.  I was just caught up in the idea that I had to do all of my imports first.  I can solve the problem by initializing the db and _then_ importing the blueprints, etc.  Will push a fix.

Reply:
[f198251](https://github.com/wikimedia/toolhunt/pull/11/commits/f198251a286998a171c62746f3228d982651a184)